### PR TITLE
New WFP cal csvs

### PR DESCRIPTION
**Pull request by rtravis-whoi**
New cal csvs for 4 coastal WFPs.
2021-01-26 - added DOFSTK cal csvs to PR.
Sheri is correct re the WFPs these are for:
12714-01
CGINS-PARADK-20525__20200928.csv
CGINS-FLORTK-00830__20200724.csv
CGINS-DOFSTK-00209__20200718.csv
12991-01
CGINS-PARADK-20541__20200928.csv
CGINS-FLORTK-01029__20200727.csv
CGINS-DOFSTK-02978__20200804.csv
12991-02
CGINS-PARADK-20447__20200928.csv
CGINS-FLORTK-01031__20200715.csv
CGINS-DOFSTK-02499__20200718.csv
12991-05
CGINS-PARADK-20542__20200928.csv
CGINS-FLORTK-01466__20200715.csv
CGINS-DOFSTK-03370__20200814.csv

### CODE REVIEW

**Comment by snwhite** [comment]
Cals appear to be for WFPs 
12714-01
12991-01
12991-02
12991-05

**Comment by rtravis-whoi** [comment]
Yes - that is correct Sheri.  My initial comment now lists the WFPs and their associated csv files.  Source files for the cal coefficients are noted in each csv.

File: calibration/FLORTK/CGINS-FLORTK-00830__20200724.csv [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+830,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_830_Calibration_2020-07-24.zip > BBFL2-830 Charsheets.pdf
+830,CC_dark_counts_cdom,47,
+830,CC_dark_counts_chlorophyll_a,49,
+830,CC_dark_counts_volume_scatter,45,
+830,CC_depolarization_ratio,0.039,
+830,CC_measurement_wavelength,700,
+830,CC_scale_factor_cdom,0.0893,
+830,CC_scale_factor_chlorophyll_a,0.0122,
+830,CC_scale_factor_volume_scatter,2.764E-06,
+830,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTK/CGINS-FLORTK-00830__20200724.csv:11]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20525__20200928.csv [hunk 2]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20525,CC_dark_offset,1.3,source_file PARAD-K_QSP2200_SN_20525_Calibration_2020-09-28.pdf
+20525,CC_scale_wet,9.56E-18,
```
Comment (snwhite) [inline calibration/PARADK/CGINS-PARADK-20525__20200928.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20541__20200928.csv [hunk 3]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20541,CC_dark_offset,1.3,source_file PARAD-K_QSP2200_SN_20541_Calibration_2020-09-28.pdf
+20541,CC_scale_wet,9.44E-18,
```
Comment (snwhite) [inline calibration/PARADK/CGINS-PARADK-20541__20200928.csv:3]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01029__20200727.csv [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+1029,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1029_Calibration_2020-07-27.zip > BBFL2-1029 Charsheets.pdf
+1029,CC_dark_counts_cdom,44,
+1029,CC_dark_counts_chlorophyll_a,48,
+1029,CC_dark_counts_volume_scatter,48,
+1029,CC_depolarization_ratio,0.039,
+1029,CC_measurement_wavelength,700,
+1029,CC_scale_factor_cdom,0.0555,
+1029,CC_scale_factor_chlorophyll_a,0.0104,
+1029,CC_scale_factor_volume_scatter,3.720E-06,
+1029,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTK/CGINS-FLORTK-01029__20200727.csv:11]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01031__20200715.csv [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+1031,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1031_Calibration_2020-07-15.zip > BBFL2-1031 CharSheets.pdf
+1031,CC_dark_counts_cdom,47,
+1031,CC_dark_counts_chlorophyll_a,48,
+1031,CC_dark_counts_volume_scatter,48,
+1031,CC_depolarization_ratio,0.039,
+1031,CC_measurement_wavelength,700,
+1031,CC_scale_factor_cdom,0.0565,
+1031,CC_scale_factor_chlorophyll_a,0.0104,
+1031,CC_scale_factor_volume_scatter,3.139E-06,
+1031,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTK/CGINS-FLORTK-01031__20200715.csv:11]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20447__20200928.csv [hunk 6]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20447,CC_dark_offset,1.3,source_file PARAD-K_QSP2200_SN_20447_Calibration_2020-09-28.pdf
+20447,CC_scale_wet,9.40E-18,
```
Comment (snwhite) [inline calibration/PARADK/CGINS-PARADK-20447__20200928.csv:3]: Looks good.

File: calibration/PARADK/CGINS-PARADK-20542__20200928.csv [hunk 7]
```diff
@@ -0,0 +1,3 @@
+serial,name,value,notes
+20542,CC_dark_offset,1.1,source_file PARAD-K_QSP2200_SN_20542_Calibration_2020-09-28.pdf
+20542,CC_scale_wet,9.52E-18,
```
Comment (snwhite) [inline calibration/PARADK/CGINS-PARADK-20542__20200928.csv:3]: Looks good.

File: calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv [hunk 8]
```diff
@@ -0,0 +1,11 @@
+serial,name,value,notes
+1466,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1466_Calibration_2019-06-17.zip > BBFL2-1466 Charsheet.pdf
```
Comment (snwhite) [inline calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv:2]: I think the source_file note should read
FLORT-K_BBFL2_SN_1466_Calibration_2020-07-15 > BBFL2-1466 Charsheets.pdf

File: calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+1466,CC_dark_counts_cdom,47,
+1466,CC_dark_counts_chlorophyll_a,48,
+1466,CC_dark_counts_volume_scatter,49,
+1466,CC_depolarization_ratio,0.039,
+1466,CC_measurement_wavelength,700,
+1466,CC_scale_factor_cdom,0.0811,
+1466,CC_scale_factor_chlorophyll_a,0.0139,
+1466,CC_scale_factor_volume_scatter,3.387E-06,
+1466,CC_scattering_angle,124,
```
Comment (snwhite) [inline calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv:11]: These are the values from 2019-06-17 calibration, not the 2020-07-15 calibration.

**Review by snwhite — CHANGES_REQUESTED** [review]
All look good, except the FLORT-K 1466 cal csv is wrong (the values are from the 2019 cal, not the 2020 cal).

File: calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv (hunk 8 above)
Comment (rtravis-whoi) [reply calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv]: updated to new vendor cal doc and cal coefficients.

**Comment by snwhite** [comment]
[PYII Instrument Cal CSVs [DT-69]](https://trello.com/c/CbhVKyjb/69-pyii-instrument-cal-csvs-dt-69)

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20200718.csv [hunk 10]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-0209,CC_frequency_offset,-7.876420e+002,source_file DOFST-K_SBE43F_SN_0209_Calibration_2020-07-18.zip > 0209.cal and SBE 43F O0209 SOCAdjusted 23Jul20.pdf
+43-0209,CC_oxygen_signal_slope,2.9150e-004,use adjusted Soc
```
Comment (snwhite) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20200718.csv:3]: All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.

File: calibration/DOFSTK/CGINS-DOFSTK-02499__20200718.csv [hunk 11]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2499,CC_frequency_offset,-8.356636e+002,source_file DOFST-K_SBE43F_SN_2499_Calibration_2020-07-18.zip > 2499.cal and SBE 43F O2499 SOCAdjusted 30Jul20.pdf
+43-2499,CC_oxygen_signal_slope,3.4847e-004,use adjusted Soc
```
Comment (snwhite) [inline calibration/DOFSTK/CGINS-DOFSTK-02499__20200718.csv:3]: All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20200814.csv [hunk 12]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-3370,CC_frequency_offset,-8.192113e+002,source_file DOFST-K_SBE43F_SN_3370_Calibration_2020-08-14.zip > 3370.cal and SBE 43F O3370 SOCAdjusted 17Aug20.pdf
+43-3370,CC_oxygen_signal_slope,2.8370e-004,use adjusted Soc
```
Comment (snwhite) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20200814.csv:3]: All values look good.  Need to be sure this slope ('adjusted Soc') value is in the instrument.

File: calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv (hunk 9 above)
Comment (snwhite) [reply calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv:11]: Updated values look good.

**Review by snwhite — APPROVED** [review]

File: calibration/DOFSTK/CGINS-DOFSTK-00209__20200718.csv [hunk 13]
```diff
@@ -0,0 +1,7 @@
...
+43-0209,CC_residual_temperature_correction_factor_a,-4.054863e-003,
+43-0209,CC_residual_temperature_correction_factor_b,2.382280e-004,
+43-0209,CC_residual_temperature_correction_factor_c,-3.440008e-006,
+43-0209,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-00209__20200718.csv:7]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02499__20200718.csv [hunk 14]
```diff
@@ -0,0 +1,7 @@
...
+43-2499,CC_residual_temperature_correction_factor_a,-3.279468e-003,
+43-2499,CC_residual_temperature_correction_factor_b,1.862367e-004,
+43-2499,CC_residual_temperature_correction_factor_c,-2.807828e-006,
+43-2499,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02499__20200718.csv:7]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20200804.csv [hunk 15]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-8.309214e+002,source_file DOFST-K_SBE43F_SN_2978_Calibration_2020-08-04.zip > 2978.cal and SBE 43F O2978 SOCAdjusted 11Aug20.pdf
+43-2978,CC_oxygen_signal_slope,2.9978e-004,use adjusted Soc
+43-2978,CC_residual_temperature_correction_factor_a,-4.150933e-003,
+43-2978,CC_residual_temperature_correction_factor_b,1.759774e-004,
+43-2978,CC_residual_temperature_correction_factor_c,-2.629011e-006,
+43-2978,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20200804.csv:7]: Looks good

File: calibration/DOFSTK/CGINS-DOFSTK-03370__20200814.csv [hunk 16]
```diff
@@ -0,0 +1,7 @@
...
+43-3370,CC_residual_temperature_correction_factor_a,-5.178602e-003,
+43-3370,CC_residual_temperature_correction_factor_b,2.443580e-004,
+43-3370,CC_residual_temperature_correction_factor_c,-3.414779e-006,
+43-3370,CC_residual_temperature_correction_factor_e,3.600000e-002,
```
Comment (cdobs) [inline calibration/DOFSTK/CGINS-DOFSTK-03370__20200814.csv:7]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-00830__20200724.csv (hunk 1 above)
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-00830__20200724.csv:11]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01029__20200727.csv (hunk 4 above)
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01029__20200727.csv:11]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01031__20200715.csv (hunk 5 above)
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01031__20200715.csv:11]: Looks good

File: calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv [hunk 17]
```diff
@@ -0,0 +1,11 @@
...
+1466,CC_angular_resolution,1.076,source_file FLORT-K_BBFL2_SN_1466_Calibration_2020-07-15.zip > BBFL2-1466 Charsheets.pdf
+1466,CC_dark_counts_cdom,48,
+1466,CC_dark_counts_chlorophyll_a,47,
+1466,CC_dark_counts_volume_scatter,48,
+1466,CC_depolarization_ratio,0.039,
+1466,CC_measurement_wavelength,700,
+1466,CC_scale_factor_cdom,0.0719,
+1466,CC_scale_factor_chlorophyll_a,0.0116,
+1466,CC_scale_factor_volume_scatter,3.039E-06,
+1466,CC_scattering_angle,124,
```
Comment (cdobs) [inline calibration/FLORTK/CGINS-FLORTK-01466__20200715.csv:11]: Looks good

File: calibration/PARADK/CGINS-PARADK-20447__20200928.csv (hunk 6 above)
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20447__20200928.csv:3]: Looks good

File: calibration/PARADK/CGINS-PARADK-20525__20200928.csv (hunk 2 above)
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20525__20200928.csv:3]: Looks good

File: calibration/PARADK/CGINS-PARADK-20541__20200928.csv (hunk 3 above)
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20541__20200928.csv:3]: Looks good

File: calibration/PARADK/CGINS-PARADK-20542__20200928.csv (hunk 7 above)
Comment (cdobs) [inline calibration/PARADK/CGINS-PARADK-20542__20200928.csv:3]: Looks good

**Review by cdobs — APPROVED** [review]
Everything looks good to me.

File: calibration/DOFSTK/CGINS-DOFSTK-02978__20200804.csv [hunk 18]
```diff
@@ -0,0 +1,7 @@
+serial,name,value,notes
+43-2978,CC_frequency_offset,-8.309214e+002,source_file DOFST-K_SBE43F_SN_2978_Calibration_2020-08-04.zip > 2978.cal and SBE 43F O2978 SOCAdjusted 11Aug20.pdf
+43-2978,CC_oxygen_signal_slope,2.9978e-004,use adjusted Soc
```
Comment (snwhite) [inline calibration/DOFSTK/CGINS-DOFSTK-02978__20200804.csv:3]: All values look good. Need to be sure this slope ('adjusted Soc') value is in the instrument.

**Pull request merged**